table/block.h

```diff
@@ -467,11 +489,25 @@ class IndexBlockIter final : public BlockIter {
   void Invalidate(Status s) { InvalidateBase(s); }
 
  private:
+  // Key is in InternalKey format
+  bool key_includes_seq_;
+  // key_includes_seq_ ? comparator_ : user_comparator_
+  const Comparator* active_comparator_;
+  BlockPrefixIndex* prefix_index_;
+  // Whether the value is delta encoded. In that case the value is assumed to be
+  // BlockHandle. The first value in each restart interval is the full encoded
+  // BlockHandle; the restart of encoded size part of the BlockHandle. The
+  // offset of delta encoded BlockHandles is computed by adding the size of
+  // previous delta encoded values in the same restart interval to the offset of
+  // the first value in that restart interval.
+  bool value_delta_encoded_;
```

Comment:
Minor: Can we pad `value_delta_encoded_` together with `key_includes_seq_`? Now it takes 8 bytes.